pycbc/inference/io/dynesty.py

```diff
@@ -24,8 +24,20 @@
 """Provides IO for the dynesty sampler.
 """
 from .base_nested_sampler import BaseNestedSamplerFile
+import hickle
```

Comment:
Why are you using this and not the function Collin wrote?

Reply:
Thanks for pointing. I fixed this issue. Also tested with a run with GW150914 and an analytical model. It seems to work now.